Assets/Scripts/Systems/Modules/Module.cs

```diff
@@ -1,64 +1,29 @@
 using System;
-using System.Collections.Generic;
 
 namespace Systems.Modules
 {
+    /// <summary>
+    ///     Holds data and root position of the module
+    /// </summary>
     [Serializable]
     public class Module
     {
         public ModuleData Data;
-        public Coordinates PivotPosition;
+        public Coordinates RootPosition;
 
         internal Module(ModuleData moduleData)
         {
             Data = moduleData;
         }
     }
-    
+
+    /// <summary>
+    ///     int substitute for vectors
+    /// </summary>
     [Serializable]
     public struct Coordinates
     {
         public int x;
         public int y;
     }
```

Comment:
Unity does have `Vector2Int`, although you'd have to make sure it gets serialized correctly